--- origsrc/webkit-sharp/samples/Makefile.am	2009-12-04 03:26:13.000000000 -0600
+++ src/webkit-sharp/samples/Makefile.am	2009-12-04 03:26:34.637731600 -0600
@@ -16,7 +16,7 @@ references=-r:../sources/$(assembly) -pk
 all: $(TARGETS)
 
 $(assembly): ../sources/$(assembly)
-	cp ../sources/$(ASSEMBLY_NAME).dll $(srcdir)
+	cp ../sources/$(ASSEMBLY_NAME).dll .
 
 funnybrowser.exe: $(assembly) $(srcdir)/FunnyBrowser.cs
 	$(CSC)  /out:$@ $(references) $(srcdir)/FunnyBrowser.cs
--- origsrc/webkit-sharp/sources/AssemblyInfo.cs.in	2009-12-04 03:26:13.000000000 -0600
+++ src/webkit-sharp/sources/AssemblyInfo.cs.in	2009-12-04 03:26:34.637731600 -0600
@@ -8,4 +8,4 @@ using System.Runtime.CompilerServices;
 [assembly: AssemblyCopyright("@ASSEMBLY_COPYRIGHT@")]
 [assembly: AssemblyVersion("@ASSEMBLY_VERSION@")]
 [assembly: AssemblyDelaySign(false)]
-[assembly: AssemblyKeyFile("@ASSEMBLY_NAME@.snk")]
+[assembly: AssemblyKeyFile("@srcdir@/@ASSEMBLY_NAME@.snk")]
--- origsrc/webkit-sharp/sources/Makefile.am	2009-12-04 03:26:13.000000000 -0600
+++ src/webkit-sharp/sources/Makefile.am	2009-12-04 03:26:34.637731600 -0600
@@ -46,10 +46,10 @@ api: get-sources
 $(API): $(srcdir)/$(RAW_API) $(srcdir)/$(METADATA)
 	cp $(srcdir)/$(RAW_API) $(API)
 	chmod u+w $(API)
-	$(GAPI_FIXUP) --api=$(srcdir)/$(API) --metadata=$(srcdir)/$(METADATA)
+	$(GAPI_FIXUP) --api=$(API) --metadata=$(srcdir)/$(METADATA)
 
 generated-stamp: $(API)
-	 $(GAPI_CODEGEN) --generate $(srcdir)/$(API) $(GTK_SHARP_CFLAGS) \
+	 $(GAPI_CODEGEN) --generate $(API) $(GTK_SHARP_CFLAGS) \
 	 	--outdir=generated --customdir=$(srcdir) \
 	 	--assembly-name=$(ASSEMBLY_NAME) && touch generated-stamp
 
